examples/todomvc/components/Footer.js

```diff
@@ -22,14 +23,12 @@ class Footer extends Component {
 
   renderFilterLink(filter) {
     const title = FILTER_TITLES[filter]
-    const { filter: selectedFilter, onShow } = this.props
 
     return (
-      <a className={classnames({ selected: filter === selectedFilter })}
-         style={{ cursor: 'pointer' }}
-         onClick={() => onShow(filter)}>
+      <RefreshLink toData={{ filter: filter }}
+                   activeCssClass='selected'>
         {title}
-      </a>
+      </RefreshLink>
```

Comment:
The Hyperlink component provided by the Navigation React plugin. It builds the href and sets the active css class.